tools/slang-unit-test/unit-test-coverage-tracing-metadata.cpp

```
@@ -144,6 +147,228 @@ SLANG_UNIT_TEST(coverageTracingMetadata)
+
+                slang::SyntheticResourceDescriptorBindingInfo descriptorInfo;
+                SLANG_CHECK(
+                    syntheticResources->getResourceDescriptorBindingInfo(i, &descriptorInfo) ==
+                    SLANG_OK);
+                SLANG_CHECK(descriptorInfo.binding == info.binding);
+                SLANG_CHECK(descriptorInfo.space == info.space);
+
+                slang::SyntheticResourceDescriptorClass descriptorClass =
+                    slang::SyntheticResourceDescriptorClass::Unsupported;
+                SLANG_CHECK(slang::getSyntheticResourceDescriptorClass(info.bindingType, &descriptorClass));
+                SLANG_CHECK(descriptorClass == slang::SyntheticResourceDescriptorClass::StorageBuffer);
```

Cppcheck (2.20.0)

[style] The function 'isBitSet' is never used.

(unusedFunction)